README.html

```diff
@@ -98,6 +98,11 @@ <h3 id="feature-wsmanplugin-modeler-plugin">WSMANPlugin Modeler Plugin</h3>
 
 <h2 id="changes">Changes</h2>
 
+<h3 id="changes-1.0.4">1.0.4</h3>
+<ul>
+    <li>Introduce zWSMANMaxElements zProperty to control the amount of returned elements per a request. (ZPS-4484)</li>
```

Comment:
I'm OK with the zProperty name, but maybe it would be better to make it consistent with what we did in WBEM: _zWBEMMaxObjectCount_.
https://github.com/zenoss/ZenPacks.zenoss.WBEM/blob/2.1.0/ZenPacks/zenoss/WBEM/__init__.py#L26